lib/facter/facter_dot_d.rb

```diff
@@ -49,7 +49,7 @@ def txt_parser(file)
       end
     end
   rescue StandardError => e
-    Facter.warn("Failed to handle #{file} as text facts: #{e.class}: #{e}")
+    Facter.warn(_("Failed to handle #{file} as text facts: #{e.class}: #{e}"))
```

Comment:
This will not create a translatable string, as it interpolates `file` and `e.class` before looking up the translation.